src/state/predictions/helpers.ts

```diff
@@ -228,6 +236,25 @@ export const getMarketData = async (): Promise<{
   }
 }
 
+export const getStats = async (): Promise<{
+  stats: Stats
+}> => {
+  const response = (await request(
+    GRAPH_API_PREDICTION,
+    gql`
+      query getMarketData {
+        market(id: 1) {
+          totalBNB
```

Comment:
In order to have a more accurate number to display, we should add `totalBNBTreasury` to the query, and subtract it from `totalBNB` to account for the treasury fee (used for the burn) + the `House` rounds

Reply:
I added totalBNBTreasury to the calculation, what are the House rounds? (failed ones?)

Reply:
@ChefKai Done, Can you review it again?